src/components/Card/Card.tsx

```diff
@@ -0,0 +1,114 @@
+import { ReactElement } from "react";
+
+import {
+  Card as MaterialCard,
+  Paper,
+  Typography,
+  CardProps,
+  styled,
+} from "@mui/material";
+import SVG from "react-inlinesvg";
+import { theme } from "../../theme";
+import { icons } from "../../icons";
+import { IconButton } from "../../IconButton";
+
+interface Props extends CardProps {
+  title: string;
+  children: ReactElement;
+  handleClose?: () => void;
+  closeButton?: boolean;
+  noPadding?: boolean;
+  warningDialog?: boolean;
+  isPinned?: boolean | null;
+  handlePin?: () => void;
+}
+
+const CardIconButton = styled(IconButton)({
+  minWidth: 0,
+  padding: 0,
+  "& svg": {
+    width: "15px",
+    height: "auto",
+  },
+  "& div": {
+    width: "30px !important",
+    height: "30px !important",
+    "&:hover": { backgroundColor: "#FFFFFF" },
+  },
+});
+
+export function Card(props: Props): JSX.Element {
+  return (
+    <MaterialCard
```

Comment:
Props here extends CardProps, do we want to pass them through to the underlying Material component?